alpha/scripts/utils/addPartnerToVoicebase.php

```diff
@@ -0,0 +1,111 @@
+<?php
```

Comment:
agreed about the special-feature permission and will remove that part but this script adds a user-role to a specific
partner, should this be done in another way?